database/src/desktop/util_desktop_macos.mm

```diff
@@ -30,8 +66,17 @@
     int retval;
     retval = mkdir(path.c_str(), 0700);
     if (retval != 0 && errno != EEXIST) return "";
-    retval = mkdir((path + "/" + app_name).c_str(), 0700);
-    if (retval != 0 && errno != EEXIST) return "";
+    // App name might contain path separators. Split it to get list of subdirs
+    std::vector<std::string> app_name_parts = split_string(app_name, '/');
+    if(app_name_parts.empty()) return "";
```

Comment:
if's have spaces after if, as in "if (app_name_parts.empty()) return "";

Reply:
Done!